node/cmd/transfer-verifier/transfer-verifier-evm.go

```diff
@@ -0,0 +1,157 @@
+package transferverifier
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/certusone/wormhole/node/pkg/telemetry"
+	txverifier "github.com/certusone/wormhole/node/pkg/transfer-verifier"
```

Comment:
It is [go convention](https://go.dev/blog/package-names) not to use dashes or underscores, e.g. guardiansigner. Please match your folders to your package name.